CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# How to contribute #
+
+All help is appreciated. All contributors are unpaid volunteers. If you really want to see something added to Exiv2 - volunteer!
+
+
+## General workflow for contributing ##
+
+1. Fork, then clone the repo:
+```
+git clone git@github.com:your-username/exiv2.git
+```
+
+2. Configure the project and build (More details about how to run CMake at XXX) :
+```
+mkdir build && cd build
+cmake .. && cmake --build . --config Release
+```
+
+3. Make sure the tests pass:
+```
+make tests			# Application tests
+./bin/unit_tests	# Unit tests
+```
+
+4. Make your change. Add tests for your change. Make the tests pass.
+5. Create the Pull Request (PR).
+	a. Suggest a reviewer if you know that some of the mantainers know about the specific change.
+    b. Check if the CI checks pass.
+    c. If your PR lives for a long time, try to avoid to press the button "Update branch" in the Pull Request view. We prefer rebasing in top of master, instead of merging master into branches.
+
+## Pull Request guidelines
+- PRs should be kept at a manageable size. Try to focus in just one goal per PR. If you find yourself doing several things that in a PR that were not expected, try to split the different tasks in different PRs.
+- Commit messages should as informative and concise as possible.
```

Comment:
i usually try to stick to the following rule:
The first line of the commit message should be < 80 characters and describe the commit briefly. If the 80 characters are too short for a summary, consider splitting the commit.
Below that, add one blank line and then a more detailed explanation if required.

This has the advantage, that `git --oneline --graph` shows a good overview and the commit messages are not wrapped on github, yet additional information is not lost.

Reply:
That is the default git recommendation, right.

Reply:
Right, I always try to do the same, but for some people this is not obvious. I will add it